README.md

```diff
@@ -239,10 +241,20 @@ solc.loadRemoteVersion('latest', function(err, solcSnapshot) {
     // An error was encountered, display and quit
   } else {
     // NOTE: Use `solcSnapshot` here with the same interface `solc` has
+    // For example:
+    const output = solcSnapshot.compile(/* ... */)
   }
 });
 ```
 
+The version should be in the long format string.
```

Comment:
"should" or "must"?

Reply:
`must`. Otherwise, it will not find the binary.